.devcontainer/Dockerfile

```diff
@@ -56,6 +56,8 @@ RUN echo "Install general purpose packages" && \
         g++ \
         g++-9 \
         gcc-9 \
+        # dependency of python services (e.g. magmad)
```

Comment:
Nit / feedback: is there any way to make this comment more clearly pertain to just iputils-ping? As it stands now I might wonder if I were a novice looking at this list.  Is it possible to `iputils-ping \ # dependency of python services (e.g. magmad)`? If not, maybe we say on line 59 `# iputils-ping is a dependency of python services (e.g. magmad)`? What do you think.

Reply:
I would have preferred the first solution, but the highlighting was not correct, so we opted for the second option.